caddyhttp/httpserver/plugin.go

```diff
@@ -447,6 +447,8 @@ var directives = []string{
 	"realip", // github.com/captncraig/caddy-realip
 	"git",    // github.com/abiosoft/caddy-git
 
+	"cache",
```

Comment:
I will leave it after log (without whitespace). Is it ok moving errors after log? So cache errors can be handled. The only problem I see is that error pages will not be compressed, but that should not be a big problem.

About rewrite having 1:many mapping, in that case rewrite should add Vary header to inform that depending on some header the response could change.